packages/Webkul/BookingProduct/src/Resources/views/admin/catalog/products/accordians/booking/event.blade.php

```diff
@@ -55,9 +55,13 @@
                     </span>
                 </div>
 
-                <div class="control-group">
-                    <label class="ticket-label">{{ __('bookingproduct::app.admin.catalog.products.special-price') }}</label>
-                    <input type="text" :name="controlName + '[special_price]'" v-model="ticketItem.special_price" class="control">
+                <div class="control-group" :class="[errors.has(controlName + '[special_price]') ? 'has-error' : '']">
+                    <label class="ticket-label">{{ __('bookingproduct::app.admin.catalog.products.special-price') }} X</label>
```

Comment:
@praem90 
why added X to the label? https://prnt.sc/v54ree

Reply:
Just added for debug purpose. I'll remove it